pkgs/applications/office/ticktick/default.nix

```diff
@@ -0,0 +1,65 @@
+{ lib, fetchurl, stdenv, wrapGAppsHook, dpkg, autoPatchelfHook,
+  glibc, gcc-unwrapped, nss, libdrm, mesa, alsa-lib, xdg-utils, systemd }:
+let
+  pname = "ticktick";
+  version = "1.0.80";
+
+  src =
+    if stdenv.hostPlatform.system == "x86_64-linux" then
```

Comment:
Isn't this already taken care by meta.platforms ?

Reply:
Was largely following `skypeforlinux`'s approach as an example of doing it with a DEB package, happy to remove it if it's not necessary.